pkg/api/controller.go

```diff
@@ -5193,3 +5198,47 @@ func extractLakeFSMetadata(header http.Header) map[string]string {
 	}
 	return meta
 }
+
+func (c *Controller) GetUsageReportSummary(w http.ResponseWriter, r *http.Request) {
```

Comment:
No RBAC level protection? Any user who can log in to the installation will get an answer for this API call?

Reply:
Should be available to any signed-in user. No special permissions required.

Reply:
@ozkatz @talSofer WDYT?